src/api/app/presenters/repositories/state_presenter.rb

```diff
@@ -4,7 +4,7 @@ class StatePresenter < SimpleDelegator
 
     def initialize(project, repository, configuration)
       download_url = configuration['download_url']
-      @download_area_exist = Backend::Api::Published.download_area_exist?(project, repository)
+      @download_area_exist = Backend::Api::Published.published_repository_exist?(project, repository)
```

Comment:
I think you should also rename the instance variable. There is not such thing as `download area` used in our terminology...

Reply:
Yeah, changing